### PR TITLE
Nodejs-Week3

### PR DESCRIPTION
**Pull request by dan-nil**
Demo of the SQL Injection [HERE](https://player.vimeo.com/video/798150283?h=82f22f94a1)

To fix the vulnerability without changing multipleStatements to false.
I decided to sanitize the query by replacing all the characters that are not letters, digits, underscore, or a space with an empty string.
In this way, two statements will not be able to pass since the first statement will not be able to be closed with ; which will throw an error.

### CODE REVIEW

File: nodejs/app.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
+const knex = require('knex')({
+    client: 'mysql2',
+    connection: {
+        host: process.env.DB_HOST || '0.0.0.0',
+        port: process.env.DB_PORT || 3306,
+        user: process.env.DB_USER || 'root',
+        password: process.env.DB_PASSWORD || '1234',
```
Comment (itsgauravsaxena) [inline nodejs/app.js]: You probably know this already and it not so important for this assignment but .env.example file instead of clear text password. ;)

**Review by itsgauravsaxena — APPROVED** [review]
Is it by mistake that same files are committed twice - both inside nodejs and week3 folder? Could be good idea to remove unnecessary files before you merge.

File: nodejs/app.js (hunk 1 above)
Comment (dan-nil) [reply nodejs/app.js]: I added now the .env.example file.

Comment (dan-nil) [reply nodejs/app.js]: I also removed the files in the wrong folder.

Thank you for the help!

**Review by itsgauravsaxena — APPROVED** [review]

**Pull request merged**